### PR TITLE
[Australia] - added conference schedule

### PR DESCRIPTION
**Pull request by swapnilogale**
----
:books: Documentation preview :books:: https://writethedocs-www--2040.org.readthedocs.build/

### CODE REVIEW

**Comment by swapnilogale** [comment]
@plaindocs the internal link error has me stumped. Help please!

**Comment by plaindocs** [comment]
LOL, you've got two links to TODO 😜

**Comment by swapnilogale** [comment]
@plaindocs those are the ones I can't find 😃

**Comment by plaindocs** [comment]
They might be in the config YAML?

**Comment by swapnilogale** [comment]
> They might be in the config YAML?

Resolved

**Comment by swapnilogale** [comment]
@plaindocs this is good to go, pending review.

File: docs/_data/australia-2023-sessions.yaml [hunk 1]
```diff
@@ -10,9 +10,7 @@
     website: 
   abstract: '<p>This moderated session will feature four experienced documentarians
     from different backgrounds sharing their impressions of what''s happening in the
-    industry today, and how it may shape our future.</p>
-
-    <p>This session would work best as the final session of the conference.</p>'
+    industry today, and how it may shape our future.</p>'
```
Comment (plaindocs) [inline docs/_data/australia-2023-sessions.yaml:13]: Did you make this change in Pretalx?

File: docs/conf/australia/2023/news/announcing-schedule.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+You've already seen our list of awesome speakers, now we can also show you the schedule so you know when they will be presenting!
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the :doc:`schedule </conf/australia/{{year}}/schedule/>` for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. 
+
+We're learning and adapting from our virtual events from the past few years, so we're introducing a moderated Q&A session after each speaker. You'll have a chance to ask most of our speakers questions after the talk via our wonderful set of moderators.
+
```
Comment (plaindocs) [inline docs/conf/australia/2023/news/announcing-schedule.rst:21]: Do you wanna talk about how to submit questions?

**Review by plaindocs — APPROVED** [review]
LGTM

**Pull request merged**